3_javascript/javascript1/week4/Homework/script.js

```diff
@@ -0,0 +1,68 @@
+let userName = "";
+const todoList = [];
+let timerId;
+
+function getReply(command) {
+  if (command.toLowerCase().includes("hello my name is")) {
+    const name = command.split(" ").slice(-1)[0];
+    userName = name;
+    return `Nice to meet you, ${userName}!`;
+  } else if (command.toLowerCase() === "what is my name") {
+    return userName
+      ? `Your name is ${userName}.`
+      : "I do not know your name yet.";
+  } else if (command.toLowerCase().startsWith("add fishing to my todo")) {
```

Comment:
why are you at some place using `startsWith` and somewhere `includes `, I think you can unify the logic